src/main/resources/R__3_3__CASELOAD_AGENCY_LOCATIONS.sql

```diff
@@ -0,0 +1,27 @@
+INSERT INTO CASELOAD_AGENCY_LOCATIONS (CASELOAD_ID, AGY_LOC_ID)
+  VALUES ('BXI', 'BXI');
```

Comment:
I'm confused. Caseload Id is clearly an agency location id?

Reply:
No it’s not I’m afraid- it’s a one to many relationship - but will discuss with you tomorrow

Reply:
No it’s not I’m afraid- it’s a one to many relationship - but will discuss with you tomorrow

Reply:
I've added some examples of 1 to many